Formula/make.rb

```diff
@@ -28,6 +28,27 @@ def install
 
     system "./configure", *args
     system "make", "install"
+
+    if build.without? "default-names"
+      (libexec/"gnubin").install_symlink bin/"gmake" =>"make"
+      (libexec/"gnuman/man1").install_symlink man1/"gmake.1" => "make.1"
+    end
+  end
+
+  def caveats
+    if build.without? "default-names" then <<-EOS.undent
```

Comment:
Changing this to:
```
    if build.without? "default-names"
      <<-EOS.undent
```
matches `findutils`. (This is a bit of an attempt to create order out of the chaos that has come to be through maintainer inconsistency throughout the years, 🙂) (Feel free to ignore this one 😄)